### PR TITLE
Update announce script for ansible-base

### PR DESCRIPTION
**Pull request by relrod**
##### SUMMARY

Change:
- s/Ansible/ansible-base/
- Update URLs

Test Plan:
- Hope for the best. (No real way to test this until release day because
  there are no files on releases.a.c for it to hash right now. A staging
  releng environment would fix this.)

Signed-off-by: Rick Elrod <rick@elrod.me>

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

hacking

### CODE REVIEW

**Comment by relrod** [comment]
This might not be perfect, but we won't fully know until release.

Also, for the beta, we'll want some manual edits to the output (since it's not a maintenance release, etc.)

File: hacking/build_library/build_ansible/announce.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # coding: utf-8
-# Copyright: (c) 2019, Ansible Project
+# Copyright: (c) 2020, Ansible Project
```
Comment (abadger) [inline hacking/build_library/build_ansible/announce.py]: I don't care but do note that the advice from Fontana is not too update the date here.  Sivel might have the page bookmarked that had that and other comments on legal boilerplate.

**Comment by abadger** [comment]
Looks good, +1 from me

File: hacking/build_library/build_ansible/announce.py (hunk 1 above)
Comment (relrod) [reply hacking/build_library/build_ansible/announce.py]: @abadger Fair, I can pull that edit out. The guideline I'm familiar with was to bump it for "substantial" changes, but I don't know if this counts as that.

**Pull request merged**